goodtablesio/plugins/github/blueprint.py

```diff
@@ -60,3 +59,29 @@ def create_job():
     tasks_chain.delay()
 
     return job_id
+
+
+# Internal
+
+def _get_owner_repo_sha(payload):
+
+    try:
+
+        # PR
+        if payload.get('pull_request'):
+            if payload['action'] not in ['opened', 'edited']:
```

Comment:
I'm just curious, what does it mean that a pull request was `edited`? Is it triggered by editing the description or adding a comment? Is it triggered by pushing new commits?